packages/http-client-js/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@typespec/http-client-js",
+  "version": "0.1.0",
+  "type": "module",
+  "private": true,
+  "main": "dist/src/index.js",
```

Comment:
does using `exports` exclusively work? just so we use more modern config

Reply:
Yeah I'll remove main